dashboard/views.py

```diff
@@ -147,7 +154,7 @@ def file_access_within_week(request, course_id=0):
 
     # get time range based on week number passed in via request
 
-    sqlString = f"""SELECT a.file_id as file_id, f.name as file_name, u.current_grade as current_grade, a.user_id as user_id
+    sqlString = f"""SELECT a.file_id as file_id, f.file_type as file_type, f.name as file_name, u.current_grade as current_grade, a.user_id as user_id
```

Comment:
This query currently throwing an error since `file_type` is not actually a DB column. The file DB schema needs to change  to fit this change. Django migrations need to be applied